README.md

```diff
@@ -1,2 +1,17 @@
-# mlkube
+# MLKube
 Machine Learning Infrastructure built on Kubernetes
+
+Kubernetes makes it possible for anyone to build container-based infrastructure.
+MLKube, an open source project, leverages the extensibility of Kubernetes to enable us to run machine learning infrastructure on top of Kubernetes.
+The primary focus of MLKube is:
+
+- Resource Management
+- Distributed ML Workloads Scheduling
+- Network
```

Comment:
We'd better explain the details of Network and Storage. I think we just focus on container network, am I right?

Reply:
This is a good point, but i have no more details about this actually.

Reply:
let's flesh this out later.